code/core/src/preview-api/modules/preview-web/UrlStore.ts

```diff
@@ -74,8 +74,10 @@ export const getSelectionSpecifierFromPath: () => SelectionSpecifier | null = ()
     const globals = typeof query.globals === 'string' ? parseArgsParam(query.globals) : undefined;
 
     let viewMode = getFirstString(query.viewMode) as ViewMode;
-    if (typeof viewMode !== 'string' || !viewMode.match(/docs|story/)) {
+    if (typeof viewMode !== 'string' || !viewMode) {
       viewMode = 'story';
+    } else if (!viewMode.match(/docs|story/)) {
+      return null;
     }
```

Comment:
What does this change have to do with the onboarding work?

This is in the preview. What other viewModes exist?
viewMode=null is not valid AFAIK in the preview?

Reply:
`settings` was being interpreted as a `viewMode`. I noticed this because the Guide page lives at `/settings/guide`.

Reply:
The settings-page renders in the manager, this code in is the preview-web; they ought to be unrelated.

Though the canvas is still visually hidden, but rendered, so perhaps you noticed an error coming from the preview/canvas whilst the settings page was rendering?

Do you recall exactly what prompted you to change this?